src/app/app.routes.ts

```diff
@@ -0,0 +1,3 @@
+import { Routes } from '@angular/router';
```

Comment:
document

          import { Routes } from '@angular/router'; // Importing Routes to define application routing

          Explantion: Adding a comment here clarifies that the imported Routes is used for defining the application's routing structure.